jax/scipy/stats/__init__.py

```diff
@@ -33,3 +33,4 @@
 from jax.scipy.stats import betabinom as betabinom
 from jax.scipy.stats import gennorm as gennorm
 from jax._src.scipy.stats.kde import gaussian_kde as gaussian_kde
+from jax._src.scipy.stats._core import mode as mode
```

Comment:
```suggestion
from jax.scipy.stats import mode
```
`jax.scipy.stats.mode` is the public import path for this function, so we should use it in the tests.

Reply:
I'm not sure to understand.
In the file  ```jax/scipy/stats/__init__.py```, the current import seems ok.
In the test file (```scipy_stats_test.py```), we do:
```
from jax.scipy import stats as lsp_stats
lax_fun = partial(lsp_stats.mode, axis=axis)
```
Which seems ok.

Reply:
Sorry, you're right. I thought this was the test file